apps/docs/pages/guides/auth/auth-user-management.mdx

```diff
@@ -0,0 +1,130 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-user-management',
+  title: 'User Management',
+  description: 'Manage your users with Supabase Auth',
+  subtitle: 'Manage your users with Supabase Auth',
+}
+
+## The User Object
+The user object stores all the information related to a user in your application. The user object can be retrieved using one of these methods:
+1. Retrieve the logged in user object using [`supabase.auth.getUser()`](/docs/reference/javascript/auth-getuser) 
+2. Retrieve a user object as an admin using [`supabase.auth.admin.getUserById()`](/docs/reference/javascript/auth-admin-listusers)
+
+A user can sign in with one of the following methods:
+- Password-based method (with email or phone)
+- Passwordless method (with email or phone)
+- OAuth
+- SAML SSO 
+
+An identity describes the authentication method that a user can use to sign in. A user can have multiple identities. These are the types of identities supported:
+- Email 
+- Phone
+- OAuth
+- SAML 
+
+<Admonition type="note">
+
+A user with an email or phone identity will be able to sign in with either a password or passwordless method (e.g. use a one-time password (OTP) or magiclink). By default, a user with an unverified email or phone number will not be able to sign in. 
+
+</Admonition>
+
+A user that uses Google OAuth and Email to sign-in will have a user object that contains the following attributes:
+```json
+{
+  "id": "00000000-0000-0000-0000-000000000000",
+  "aud": "authenticated",
+  "role": "authenticated", // the role used by Postgres to perform Role Level Security (RLS) checks 
```

Comment:
Consider adding a description of each claim below or elsewhere

Reply:
do you think it would still be useful to add an example of the JSON object in the docs?

Reply:
I think it might help visually  if the description of the claims are in table form. Think it wouldn't be a big loss to do w/o it as well though

Reply:
converted into a table - thanks for the feedback, it definitely looks alot more organised now